src/Microsoft.OData.Client/DataServiceContext.cs

```diff
@@ -2923,7 +2923,7 @@ protected internal Type DefaultResolveType(string typeName, string fullNamespace
         /// <param name="response">The response of previous page</param>
         /// <param name="entity">The entity that contains the property to load.</param>
         /// <param name="propertyName">The name of the property of the specified entity to load.</param>
-        /// <returns>An instance of <see cref="T:Microsoft.OData.Client.QueryOperationResponse`1" /> that contains the results of the request.</returns>
+        /// <returns>An instance of <see cref="Microsoft.OData.Client.QueryOperationResponse`1" /> that contains the results of the request.</returns>

```

Comment:
I didn't resolve ALL warnings. I will split this across multiple pull requests, as otherwise it will get too big to review. There will still be warnings left after this pull request is completed. Either way, your comment is definitely valid, I added the change.